Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM node:latest
 
-ADD . /app
+RUN mkdir -p /app
 WORKDIR /app
 
+COPY package.json /app/
 RUN npm install
+
+COPY . /app
```

Comment:
wouldn't this override node_modules if you install them locally before running docker?